patches/lief/0001-fix-change-variable-types-to-prevent-64-bit-to-32-bit-conversion.patch

```diff
@@ -0,0 +1,119 @@
+From 2bc95fb02ccdb6b5c8f6859ae383184867d95679 Mon Sep 17 00:00:00 2001
```

Comment:
We can get rid of this patch when https://github.com/postmanlabs/postject/pull/23 lands.